versions/src/main/java/cc/polyfrost/oneconfig/utils/commands/PlatformCommandManagerImpl.java

```diff
@@ -154,23 +152,55 @@ private CommandManager.Pair<String[], CommandManager.InternalCommand> getCommand
                     // work backwards to find the first match
                     CommandManager.InternalCommand command = get(root, argsIn);
                     if (command != null) {
+                        String primaryPath = command.getPrimaryPath()
```

Comment:
are you sure this doesn't abuse primary paths? have you checked if aliases still work?

Reply:
This works with `/oneconfig profiles <overloaded main arg>`, `/oneconfig profile <overloaded arg>`, `/ocfg profile <overloaded arg>`, so :+1: